dubbo-test/dubbo-test-check/src/main/resources/META-INF/services/org.junit.platform.launcher.TestExecutionListener

```diff
@@ -1 +1,5 @@
+# DubboRegistryCenterStarted should be the first one because of the startup of zookeeper
+org.apache.dubbo.test.check.RegistryCenterStarted
```

Comment:
> Pls add JVM arg to disable embedded zk server, allow users to use external zk server.

It's done.

Reply:
> When enable thread checking by `-DcheckThreads=true`, `DubboTestChecker` will dump threads after finished each test class. The embedded zk server will also create some threads, such as creating a new netty thread when establishing a connection, which will interfere with the thread check of dubbo test. Is it possible to start zk server in a separate process?

Now, it can use a dependent process to start zookeeper server.